abel/tools/vmi.py

```diff
@@ -441,7 +441,7 @@ def toPES(radial, intensity, energy_cal_factor, per_energy_scaling=True,
         The units affect the units of the output. e.g. inputs in
         eV/pixel\ :sup:`2`, will give output energy units in eV.  A value of
         :math:`1.148427\times 10^{-5}` cm\ :math:`^{-1}/`\ pixel\ :sup:`2`
-        applies for "examples/data/O-ANU1024.txt" (with Vrep = −98 volts).
+        applies for "examples/data/O2-ANU1024.txt" (with Vrep = −98 volts).
```

Comment:
No, the discussion and  scaling factor actually apply to O<sup>-</sup> photodetachment, while, in the past, the data for `example_hansenlaw.py` was changed to `O2-ANU1024.txt`, no doubt to reduce the PyAbel package size. I will update the `abel.tools.vmi.toPES` text during today.

Reply:
OK, now I see that it was indeed `O-` but then was replaced by `O2-` here: ff35dbf. Perhaps then all the references to it should be adjusted accordingly, or maybe `SampleImage(n=..., name='Ominus')` can be used instead?

On the other hand, we actually don't include any example data in the package, so it shouldn't be a problem to readd that file back, if it's useful at all.

One strange thing that I've just noticed is that the PyPI package does include the `examples` directory, although without `examples/data` and even without `examples/README.txt`, but the conda packages don't include any examples (which is probably expected). @DanHickstein, what do you think we should do about this? I guess, providing `pyabel-examples` as a separate package (with the data files) might be useful, but I have no idea how this can be done easily...

Reply:
Thanks for catching the typos!

I think that it's okay if we don't install the examples in the case of PyPi or Conda install, since most users will be interacting with the examples as part of the docs. I'm actually a bit surprised that they are installed with the PyPi install, since I think that we previously observed that they were not installed:
https://github.com/PyAbel/PyAbel/issues/26#issuecomment-170242549
But I don't have strong feelings on the issue.

What is the motivation for moving the examples to a separate repo? I think that the standard practice is to keep them in the same repository. For example, matplotlib has them in the main repo: https://github.com/matplotlib/matplotlib/tree/main/galleries

Reply:
Here is a diff for `abel/tools/vmi.py` with the corrected constants for O<sub>2</sub><sup>-</sup>:

```
diff vmi.py ~/Downloads/.
422,424c422,424
<     generated energy scale with published spectra. e.g. for O\ :sub:`2`\ :sup:`−`
<     photodetachment, the origin band occurs at the
<     electron affinity :math:`EA = 3,613` cm\ :math:`^{-1}`. Values for
---
>     generated energy scale with published spectra. e.g. for O\ :sup:`−`
>     photodetachment, the strongest fine-structure transition occurs at the
>     electron affinity :math:`EA = 11\,784.676(7)` cm\ :math:`^{-1}`. Values for
443,444c443,444
<         :math:`1.204\times 10^{-5}` cm\ :math:`^{-1}/`\ pixel\ :sup:`2`
<         applies for "examples/data/O2-ANU1024.txt" (with Vrep = −2200 volts).
---
>         :math:`1.148427\times 10^{-5}` cm\ :math:`^{-1}/`\ pixel\ :sup:`2`
>         applies for "examples/data/O2-ANU1024.txt" (with Vrep = −98 volts).
458c458
<         is :math:`10^{7}/\lambda` (nm) e.g. ``1.0e7/454.5`` for
---
>         is :math:`10^{7}/\lambda` (nm) e.g. ``1.0e7/812.51`` for
466c466
<         is applied. e.g. ``-2200`` (volts), for "examples/data/O2-ANU1024.txt".
---
>         is applied. e.g. ``-98`` (volts), for "examples/data/O2-ANU1024.txt".
```

Reply:
Thanks, @stggh! I've incorporated your corrections (with minor formatting changes, to avoid line breaks between “cm” and “⁻¹”), please check that now they all are as expected.